magstock/__init__.py

```diff
@@ -34,9 +34,17 @@ def PREREG_DONATION_OPTS(self):
 
 @Session.model_mixin
 class SessionMixin:
+    def food_consumers(self):
+        """
+        :return: list of anyone who gets food, regardless of if they're comp'd food or if they paid for it
+        """
+        return [a for a in self.query(Attendee).all() if a.gets_food]
```

Comment:
btw this slightly sucks that in order to use a DB query here, we have to rewrite a bunch of the existing logic for ```.gets_food``` as sqlalchemy statements.  since the amount of attendees in magstock will be less than 600 and this   is for a not often used report, I took the dumb/easy/lame way out and just do the filtering in python.